frontend/src/common.ts

```diff
@@ -366,9 +366,11 @@ export interface LoginMetrics {
     logins_per_day: number;
     percent_active: number;
     percent_inactive: number;
-    total_users: number;
+    total_students: number;
```

Comment:
I added a comment on `dashboard.go` about updating this to be `total_residents` while the thought is fresh. If you can update it, that'd be great.